app/grid.scss

```diff
@@ -1,74 +1,209 @@
+@font-face {
+    font-family: Fontin Regular;
+    src: url(fonts/fontin2_pc/Fontin-Regular.ttf);
+}
+
+@font-face {
+    font-family: Muli;
+    src: url(fonts/Muli/Muli-Regular.ttf);
+}
+
+
 body {
     margin: 0;
     height: 100%;
     width: 100%;
+    font-family: Arial, Helvetica, sans-serif;
 }
 
 .grid {
     display: block;
+}
+
+#main {
     color: #fff;
-    font-size: 150%;
+    font-size: 100%;
     height: 100%;
     width: 100%;
     margin: 0;
 }
 
 .menu {
-    background-color: rgb(197, 48, 48);
+    background-color: rgb(255, 255, 255);
     height: 5%;
 }
 
-.header {
-    background-color: #444;
-    height: 5%;
+#search-bar {
+    text-align: right;
+    font-size: 30px;
+    margin: 0 auto;
+    color: rgb(73, 73, 73);
 }
 
 .profile {
-    background-color: rgb(51, 112, 27);
+    background-color: rgb(16, 13, 16);
     height: 15%;
 }
 
+.profile .grid {
+    margin-top: 110px;
+}
+
+#name {
+    font-size: 27px;
+    font-weight: bold;
+    margin-left: 10px;
+    padding-top: 20px;
+}
+
+#title {
+    margin-left: 10px;
+    font-size: 13px;
+    font-weight: bold;
+}
+
+#links {
+    margin-left: 10px;
+}
+
+.link {
+    padding: 3px;
+    font-size: 20px;
+}
+
+#picture {
+    margin: 15px;
+    border: 3px solid #ffffff;
+}
+
+#picture {
+    height: 100px;
+    width: 100px;
+    background-color: rgb(216, 21, 21);
+    border-radius: 100%;
+}
+
+#bio {
+    margin: 10px;
+    padding-top: 20px;
+}
+
 .posts {
-    background-color: rgb(70, 230, 241);
-    height: 70%;
+    background-color: rgb(255, 255, 255);
+    margin: 0px;
+    list-style-type: none;
+}
+
+.post {
+    text-align: center;
+    background-color: rgb(255, 255, 255);
+    color: grey;
+    width: 80%;
+    height: 150px;
+    margin: 30px auto;
+    border: 1px solid #b1b1b1;
+    border-radius: 10px;
+    padding: 10px;
+    box-shadow: 0px 0px 1px 1px #d3d3d3;
+}
+
+#time-line {
+    text-align: left;
+    font-family: Sans;
+}
+
+
+#diamond {
+    font-size: 12px;
+}
+
+h2 {
```

Comment:
This should be higher in the CSS file, as usually you define stuff in this order:

* font-faces
* general rules on elements (body, p, h1, h2, etc.)
* classes
* ids
* media queries